### PR TITLE
Database client test

### PR DESCRIPTION
**Pull request by nutrina**
This fixes https://github.com/gitcoinco/passport/issues/616

### CODE REVIEW

**Comment by farque65** [comment]
This PR addresses #573

**Review by farque65 — APPROVED** [review]
LGTM

File: database-client/jest.config.cjs [hunk 1]
```diff
@@ -0,0 +1,16 @@
+/** @type {import('ts-jest/dist/types').InitialOptionsTsJest} */
+// module.exports = {
+//   preset: 'ts-jest',
+//   testEnvironment: 'node',
+// };
```
Comment (farque65) [inline database-client/jest.config.cjs:5]:
```suggestion
```

**Review by aminah-io — APPROVED** [review]
Tests pass locally, everything looks good, LGTM 🚀!

**Pull request merged**